Adyen/Core/Payment Methods/BLIKPaymentMethod.swift

```diff
@@ -8,17 +8,22 @@ import Foundation
 
 /// A BLIK payment method.
 public struct BLIKPaymentMethod: PaymentMethod {
-
-    /// :nodoc:
+    
     public let type: PaymentMethodType
 
-    /// :nodoc:
     public let name: String
+    
+    public var merchantProvidedDisplayInformation: MerchantCustomDisplayInformation?
 
     /// :nodoc:
     public func buildComponent(using builder: PaymentComponentBuilder) -> PaymentComponent? {
         builder.build(paymentMethod: self)
     }
+    
+    /// :nodoc:
+    public func defaultDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation {
+        DisplayInformation(title: name.uppercased(), subtitle: nil, logoName: type.rawValue)
```

Comment:
`name.uppercased()` - was this a requirement before? 🤔
Right now it is `Blik`, not a `BLIK`

Reply:
that's how it was, I just refactored where the `uppercased()` is called :-)

Reply:
OK, I believe `uppercased` is correct